src/main/java/org/logstash/filters/geoip/Fields.java

```diff
@@ -44,7 +45,14 @@ enum Fields {
   LOCATION("geo.location", "location"),
   LATITUDE("geo.location.lat", "latitude"),
   LONGITUDE("geo.location.lon", "longitude"),
-  ORGANIZATION("mmdb.organization", "organization");
+  ORGANIZATION("mmdb.organization", "organization"),
+  NETWORK("traits.network", "network"),
```

Comment:
Couldn't find any matching names on the ECS schema. I've ended up using the `treats` prefix, as those fields are related to the IP's address characteristics, and are not about the connection, network, etc.
Another option that crossed my mind was using the `is_` prefix, for example `is_anonymous_vpn` , which I didn't implement to keep it similar to the ES geoip field names. 

Do you have any suggestions on this?

Reply:
Looking at the ECS guideline for [custom fields](https://www.elastic.co/guide/en/ecs/current/ecs-custom-fields-in-ecs.html#_proper_names) and the purpose of Anonymous IP database, the name comes across me is `anonymous`.

`anonymous.network`, `anonymous.hosting_provider` ...

Reply:
Yes, I've initially named them using the `anonymous`/`anom` prefix, but TBH I wasn't convinced considering those fields are actually IP's characteristics/traits, for example:
The `network` field is the IP's CIDR ( e.g:  `74.209.16.0/20`), and is more about the network than anonymity.
The other fields are basically traits of the requested IP address, and indicates if the IP **belongs**/**is** a VPN, Tor, public proxy, or hosting provider, etc. Finally, I ended up using the same name as MaxMind does ([traits](https://github.com/maxmind/GeoIP2-java/blob/main/src/main/java/com/maxmind/geoip2/record/Traits.java)), so we could group these and future IP's characteristics fields under this prefix. WDYT?

Reply:
right, `anonymous` doesn't sound a fit. My main concern is neither `anonymous` nor `traits` hint me this is about IP/ network. How about `ip_traits`?

Reply:
Done! could you please have a second look (https://github.com/logstash-plugins/logstash-filter-geoip/pull/223/commits/75a85bc1634f28335eec4cbfe4c878a32fa52ba7)? In addition to that, I've downgraded back to 2.x the MaxMind libraries (https://github.com/logstash-plugins/logstash-filter-geoip/pull/223/commits/5b1a40009a68d7b12cdc2f8d76eca73f8c8ad8ad), so we don't need to enforce Java 11, avoiding compatibility issues with LS 7 & Java 8.

Reply:
@edmocosta 

It seems that there is no code using `Field.NETWORK`. How should I configure it to use `ip_traits.network`?